pydatastructs/linear_data_structures/algorithms.py

```diff
@@ -722,3 +723,67 @@ def partition(low, high, pick_pivot_element):
         array._modify(force=True)
 
     return array
+
+def longest_common_subsequence(seq1, seq2) -> tuple:
+    """
+    Implements Longest Common Subsequence
+
+    Parameters
+    ========
+
+    seq1: String or List or Tuple
+    seq2: String or List or Tuple
+
+    Returns
+    =======
+
+    output: tuple
+    (Length of LCS, Common Sequence)
+    Common Sequence will be of the same data type as seq1.
+
+    Examples
+    ========
+
+    >>> from pydatastructs import longest_common_subsequence as LCS
+    >>> LCS("ABCDEF", "ABBCDDDE")
+    (5, 'ABCDE')
+    >>> arr1 = ['A', 'P', 'P']
+    >>> arr2 = ['A', 'p', 'P', 'S', 'P']
+    >>> LCS(arr1, arr2)
+    (3, ['A', 'P', 'P'])
+
+    References
+    ==========
+
+    .. [1] https://en.wikipedia.org/wiki/Longest_common_subsequence_problem
+    """
+    if not(isinstance(seq1, (str, tuple, list))):
```

Comment:
A blank line after 759 and it isn't necessary to use brackets with no.
It should be
`if not isinstance (...)`

Reply:
Can you just find all the issue and report me, as Gagandeep asked me to reduce the commits to reduce the resource used by travis

Reply:
I will change this for now.

Reply:
> Can you just find all the issue and report me, as Gagandeep asked me to reduce the commits to reduce the resource used by travis

Issues can only be reported when they arise. Can't report them before their occurance.

Commits can be squashed into one so no worries.